rclcpp/src/rclcpp/executor.cpp

```diff
@@ -268,7 +268,8 @@ Executor::add_node(rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_pt
   // If the node already has an executor
   std::atomic_bool & has_executor = node_ptr->get_associated_with_executor_atomic();
   if (has_executor.exchange(true)) {
-    throw std::runtime_error("Node has already been added to an executor.");
+    throw std::runtime_error(
+      std::string("Node (%s) has already been added to an executor.", node_ptr->get_name()));
```

Comment:
how about telling fully qualified name including namespace?

```suggestion
      std::string("Node (%s) has already been added to an executor.", node_ptr->get_fully_qualified_name()));
```